modules/oc-mirror-operator-catalog-filtering.adoc

```diff
@@ -181,7 +178,9 @@ mirror:
           - name: stable
             maxVersion: 6.0.0
 ----
-|Within the selected channel of that package, all versions up to the `maxVersion` (not relying on the shortest path from the upgrade graph). The head of the channel is not included, even if multiple channels are included in the filtering.
+|Within the selected channel of that package, all versions up to `maxVersion`.
+Head of channel is not included, even if multiple channels are included in the filtering.
+You should expect errors if this filtering leads to a channel with multiple heads. The `defaultChannel` should be used in case the filtered channels are not the default.
```

Comment:
```suggestion
You might see errors if this filtering leads to a channel with multiple heads. You must use the `defaultChannel` field in case the filtered channels are not the default.
```

Same comment